mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule DockerCompose.MixProject do
     [
       app: :docker_compose,
       version: "0.3.1",
-      elixir: "~> 1.10",
+      elixir: "~> 1.15",
```

Comment:
Do we need to require Elixir 1.15? For a library it's generally better to keep this requirement as loose as possible. Did you take advantage of any 1.15 features?

Reply:
I did not. I will keep it at `1.10` then.

Reply:
Force-pushed. [Diff](https://github.com/altworx/docker-compose-elixir/compare/d29dc1ddb35d24b2dc2f31a02836e62a43ddc025..71679bcaeffcfbe0ae79f435a5a171cb784ffd23).

Reply:
Thanks!